src/h5web/visualizations/shared/VisCanvas.tsx

```diff
@@ -7,7 +7,7 @@ import { computeVisSize } from './utils';
 import AxisSystem from './AxisSystem';
 import AxisSystemProvider from './AxisSystemProvider';
 
-const AXIS_OFFSETS = { vertical: 72, horizontal: 36, fallback: 16 };
+const AXIS_OFFSETS = { vertical: 72, horizontal: 54, fallback: 16 };
```

Comment:
I had to make a bit of space for the axis label. Otherwise, it was superimposed with ticks.

Reply:
Any way of making this number dynamic based on the absence/presence of a label?

Reply:
I thought about it but then I asked myself the following question: it is better to make space when we need to or to keep a consistent layout ?

Reply:
Excellent question... Hmm, so the layout will change when switching from an NXdata group to a classic dataset. I don't think it's a big deal. I feel like we should take more space if we can, especially since Nexus visualizations will also have a title... It will end up being quite a bit of unused whitespace for classic visualizations.

Reply:
Happy to deal with this later either way.

Reply:
Yeah perhaps we should leave it for once we tackled #244 to make everything in one go.

Reply:
Well, I looked into it and in the end, it was quite simple to implement : https://github.com/silx-kit/h5web/pull/260/files#diff-1b754f8c9c4a57ec8444d387642a4309b20ca9327a9621a7c56a33c731f7d1fbR26